fnzksxl/DP/등굣길.py

```diff
@@ -0,0 +1,21 @@
+def solution(m, n, puddles):
+    dp = [[0 for i_ in range(m+1)] for j in range(n+1)]
```

Comment:
```suggestion
     dp = [[0]*(m+1) for _ in range(n+1)]
```
어느쪽이든 상관없지만 저는 이렇게 작성합니다!

Reply:
리스트 컴프레헨션으로 쓸지 안 쓸지는 뭐랄까 그 날의 기분에 따라 달린 느낌이랄까요